setup.cfg

```diff
@@ -35,6 +35,8 @@ universal = 1
 max-line-length = 99
 exclude = docs, .tox, .git, __pycache__, .ipynb_checkpoints
 ignore = W503
+per-file-ignores = 
+	ctgan/data.py:T001
```

Comment:
Do we really want to keep the `print` statements there or use `LOGGER` instead ?